src/shared/components/lock-screen/LockScreen.js

```diff
@@ -42,7 +42,9 @@ class LockScreen extends Component {
 
         input.focus();
         input.addEventListener('keydown', this.handleKeyboardInput);
-        input.addEventListener('blur', () => input.focus());
```

Comment:
Why was this changed?

Reply:
For testing purposes, I had to log this arrow function. That is the reason for that block being there. I'll change it back to the original form